bottlecap/src/metrics/enhanced/lambda.rs

```diff
@@ -573,15 +558,25 @@ impl Lambda {
 
         tokio::spawn(async move {
             // get list of all process ids
-            let pids = proc::get_pid_list();
+            let mut pids = proc::get_pid_list();
 
-            // Set fd_max and initial value for fd_use to -1
+            // Set fd_max and initial value for fd_use
             let fd_max = proc::get_fd_max_data(&pids);
-            let mut fd_use = -1_f64;
+            let mut fd_use = if let Ok(fd_use) = proc::get_fd_use_data(&pids) {
+                fd_use
+            } else {
+                debug!("Could not get file descriptor use.");
```

Comment:
More descriptive? Which purpose?
```suggestion
                debug!("Could not get file descriptor use.");
```